### PR TITLE
Fix resize_nodes.go test.

### LINKED ISSUE #14695: flaky e2e: Network when a minion node becomes unreachable [replication controller] recreates pods scheduled on the unreachable minion node AND allows scheduling of pods on a minion after it rejoins the cluster
Nodes Network when a minion node becomes unreachable [replication controller] recreates pods scheduled on the unreachable minion node AND allows scheduling of pods on a minion after it rejoins the cluster(Failed 17 times in the last 30 runs. Stability: 43 %)Failed with 0 retries

### PR DESCRIPTION
**Pull request by wojtek-t**
This should fix #14695

@gmarek @fgrzadkowski

### CODE REVIEW

File: test/e2e/util.go [hunk 1]
```diff
@@ -768,7 +768,10 @@ func waitForPodToDisappear(c *client.Client, ns, podName string, label labels.Se
 // In case of failure or too long waiting time, an error is returned.
 func waitForRCPodToDisappear(c *client.Client, ns, rcName, podName string) error {
 	label := labels.SelectorFromSet(labels.Set(map[string]string{"name": rcName}))
-	return waitForPodToDisappear(c, ns, podName, label, 20*time.Second, 5*time.Minute)
+	// NodeController evists pod after 5 minutes, so we need timeout greater than that.
+	// Additionally, there can be non-zero grace period, so we are setting 10 minutes
+	// to be on the safe size.
+	return waitForPodToDisappear(c, ns, podName, label, 20*time.Second, 10*time.Minute)
```
Comment (gmarek) [inline test/e2e/util.go]: 10 minutes might be an overkill, but better safe than sorry.

**Comment by gmarek** [comment]
LGTM

File: test/e2e/util.go [hunk 2]
```diff
@@ -768,7 +768,10 @@ func waitForPodToDisappear(c *client.Client, ns, podName string, label labels.Se
 // In case of failure or too long waiting time, an error is returned.
 func waitForRCPodToDisappear(c *client.Client, ns, rcName, podName string) error {
 	label := labels.SelectorFromSet(labels.Set(map[string]string{"name": rcName}))
-	return waitForPodToDisappear(c, ns, podName, label, 20*time.Second, 5*time.Minute)
+	// NodeController evists pod after 5 minutes, so we need timeout greater than that.
```
Comment (fgrzadkowski) [inline test/e2e/util.go]: s/evists/evicts

Comment (wojtek-t) [inline test/e2e/util.go]: Done.

**Comment by wojtek-t** [comment]
@fgrzadkowski - PTAL

**Comment by fgrzadkowski** [comment]
LGTM

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Pull request merged**